### PR TITLE
improve rwd of header

### LINKED ISSUE #4121: Environment Select shouldn't appear on newline if there is enough space
The select component in the navigation bar shouldn't go onto a new line if there is more than enough space to display it besided the Inmanta Logo

### PR DESCRIPTION
**Pull request by matborowczyk**
# Description

* Short description here *

closes #4121 

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [x] Attached issue to pull request
- [x] Changelog entry
- [x] Code is clear and sufficiently documented
- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Correct, in line with design
~~- [ ] End user documentation is included or an issue is created for end-user documentation (add ref to issue here: )~~

# Reviewer Checklist:

- [x] Sufficient test cases (reproduces the bug/tests the requested feature)
- [x] Code is clear and sufficiently documented
- [ ] Correct, in line with design

### CODE REVIEW

File: changelogs/unreleased/4121-top-nav-rwd.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+change-type: patch
+description: improve rwd of header in range of 768px - 1200px
+issue-nr: 4119
```
Comment (LauraStordeur) [inline changelogs/unreleased/4121-top-nav-rwd.yml:3]: number not aligned with branch number, ;) should be 4121

File: src/UI/Root/Components/Header/Header.tsx [hunk 2]
```diff
@@ -34,4 +34,24 @@ const StyledImage = styled(SimpleBackgroundImage)`
 
 const StyledHeader = styled(PageHeader)`
   background-color: transparent;
+  //var(--pf-global--breakpoint--md) doesn't seems work
```
Comment (LauraStordeur) [inline src/UI/Root/Components/Header/Header.tsx:37]: but isn't this a value you need to assign to a css prop?

**Comment by LauraStordeur** [comment]
screenshot / ticket number in description / check your boxes ;)

File: src/UI/Root/Components/Header/Header.tsx (hunk 2 above)
Comment (matborowczyk) [reply src/UI/Root/Components/Header/Header.tsx:37]: but that is patternfly variable, it should be available out of the box as other variables used in the project

**Comment by matborowczyk** [comment]
Screenshot 2022-10-24 at 12 10 03

**Review by LauraStordeur — APPROVED** [review]

**Pull request closed without merging**